src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6139,7 +6139,7 @@ public class TypeDelegator : System.Reflection.TypeInfo
     public delegate bool TypeFilter(Type m, Object filterCriteria);
     public abstract partial class TypeInfo : System.Type, System.Reflection.IReflectableType
     {
-        internal TypeInfo() { }
+        protected TypeInfo() { }
```

Comment:
@danmosemsft raised this issue so I'll let him decide which is more important: making these three RefEmit types line up or keeping the door closed on user-subclassing of TypeInfo (the door is already open on System.Type, so the worms are pretty much out of the can, but it is technically a new api so it needs approval.)